webapp/tests/mocha/unit/enketo/medic-xpath-extensions.spec.js

```diff
@@ -102,14 +132,48 @@ describe('medic-xpath-extensions', function() {
     });
   });
 
-  describe('#to-bikram-sambat()', () => {
+  describe('#difference-in-weeks', function () {
```

Comment:
Is there a reason not to add test cases with two dates crossing over a leap year and when the end date is before the start date?

Reply:
The dateDiff function has been refactored extensively; these unit tests now seem to test the moment library itself more than the surrounding code. Therefore, the behavior should be consistent. I've added the tests regardless.